### DISCUSSION TITLE
Report your working camera/DVR here!

### DISCUSSION THREAD

**Discussion by toxuin** [toxuin/alarmserver#17] [General]
I'll start! I've tested it working with the following cams:
- Misecu 5MP Wifi AI Cam - hisilicon server, FTP server
- Uniden U-Bell DB1 doorbell - hikvision server in raw tcp mode
- 3xLogic VX-2M-2D-RIA - hikvision server

**Comment by tomdudu38** [comment]
- noname (or lot of names) camera '(icsee) 5MP WIFI MODEL A8-5MP-EU (xiongmai) hisilicon 
  https://tse4.explicit.bing.net/th?id=OIP.OAgO7ViVADNhgWqpR-AlzgHaHa
  https://ae01.alicdn.com/kf/A594da84fc010449ea008955350a98629d.jpg
